interface/src/app/gene/gene-network.tpl.html

```diff
@@ -0,0 +1,10 @@
+<div>
+  <h1 class="page-header">
+    Gene Network
+  </h1>
+  <div class="well">
+    <geneset-search-form ng-if="userObj"></geneset-search-form>
```

Comment:
Right - I was copying the behavior that the GIANT webserver uses for searching genes and/or Tribe genesets (the Tribe geneset section also only appears if the user is logged in using Tribe). I see what you mean though, so maybe it would be worth it to disable the section instead of removing it.